content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -11,12 +11,18 @@ This minimal install provides traffic management features of Istio.
 
 ## Prerequisites
 
-Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites) described in the Quick Start guide.
+Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites) described in the Quick Start guide.  The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.
```

Comment:
Please be careful how you reference other documents. This link is difficult to maintain because you place the document and the section of the document far apart in the text and the document isn't even part of the link. If the title of the document changes, someone will have to find the reference in the text. also be mindful of the 80 character line length to make reviewing easier and for consistency.
```suggestion
Refer to the [Kubernetes Installation prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites) for more information.  The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.
```